fpo-api/fpo_api/settings.py

```diff
@@ -224,7 +224,7 @@
     OIDC_RP_AUTHENTICATION_REDIRECT_URI = (
         os.getenv("OIDC_RP_AUTHENTICATION_REDIRECT_URI", "/family-law-act/")
     )
-    OIDC_RP_KC_IDP_HINT = os.getenv("OIDC_RP_KC_IDP_HINT")
+    OIDC_RP_KC_IDP_HINT = os.getenv("OIDC_RP_KC_IDP_HINT", "bceid")
```

Comment:
We don't want to set a default here.  We'll set the variable as necessary in the deployment environment.